spec/firebase_app_distribution_auth_client_spec.rb

```diff
@@ -29,87 +29,101 @@
       .and_return(fake_binary_contents)
     allow(fake_binary_contents).to receive(:key)
       .and_return("fake_service_key")
-    allow(File).to receive(:exist?).and_return(false)
-
-    allow(ENV).to receive(:[])
-      .with("GOOGLE_APPLICATION_CREDENTIALS")
-      .and_return(nil)
-    allow(ENV).to receive(:[])
-      .with("FIREBASE_TOKEN")
-      .and_return(nil)
-    allow(ENV).to receive(:[])
-      .with("XDG_CONFIG_HOME")
-      .and_return(nil)
   end
 
   describe '#fetch_auth_token' do
-    it 'uses service credentials for authorization when the path is passed in' do
-      expect(auth_helper.fetch_auth_token("google_service_path")).to eq("service_fake_auth_token")
-    end
+    describe 'with all other auth variables as nil or empty' do
+      [nil, ""].each do |testval|
```

Comment:
Can you give `testval` a more meaningful name, maybe `nothing_value` or something?